src/crystal/system/unix/username.cr

```diff
@@ -0,0 +1,11 @@
+require "c/pwd"
+
+module Crystal::System
+  def self.current_user_name
+    if (pwd = getpwuid(LibC.getuid))
```

Comment:
```suggestion
    if pwd = getpwuid(LibC.getuid)
```

Reply:
Ah yeah, this is a habit from standardrb where it makes you wrap assignments in conditionals in parens to make it more of a clear indication that it's intentional and not accidentally forgetting to make it `==`. I removed it though in a fixup if that's not a desired style in the Crystal stdlib